dev/src/Command/AddComponentCommand.php

```diff
@@ -111,17 +123,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ->render();
         }
 
-        $productHomepage = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product homepage? ')
-        );
-
-        $productDocumentation = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product documentation URL? ')
-        );
+        $version = basename(dirname($proto));
+        // yaml file contains documentation_uri and
+        // it is available at the same location as proto file
+        $libraryName = strtolower($new->componentName);
+        $yamlFileName = $libraryName . '_' . $version . '.yaml';
+        $yamlFilePath = dirname($proto) . '/' . $yamlFileName;
+        $googleApisDir = explode('/google/', $yamlFilePath)[0];
+        $productDocumentation = null;
+        if (file_exists($yamlFilePath)) {
+            $yamlFileContent = file_get_contents($yamlFilePath);
+            preg_match('/(?i)^.*\bdocumentation_uri\b.*$/m', $yamlFileContent, $matches);
+            $productDocumentation = explode('documentation_uri: ', $matches[0])[1];
+            $productHomepage = explode('/docs', $productDocumentation)[0];
```

Comment:
Are we sure this is reliable? I am not sure it's always the case, but if we can show that it is true for all existing values of `documentation_uri`, then I'm on board with it.

We could do a quick `$http` check to see, and if it's a 200 we could add it, and if it's a 404 we could prompt for it.

Reply:
I checked all the v1 yaml files and got a mix of `200`, `301`, `302` and `404`. I only got `404` for `https://cloud.google.com/redis`, `https://cloud.google.com/securitycentermanagement`. In both the cases, doc page is also 404.

So, yeah, this works whenever docs page works.

So, we can have a 200 check on this for to be on the safe side then. (I don't expect `301` and `302` for the new libraries anyway)